app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -38,22 +22,26 @@ def release_pdf?(_form526_submission)
 
     private
 
-    def generate_pdf(_form526_submission, _assessed_data)
-      raise 'method not implemented yet' # RuntimeError
+    def query_and_assess_lighthouse(form526_submission)
+      client = lighthouse_client(form526_submission)
+      medications = assess_medications(client.list_resource('medication_requests'))
+      { medications: medications }
+    end
+
+    def assess_medications(medications)
+      return [] if medications.blank?
+
+      RapidReadyForDecision::LighthouseMedicationRequestData.new(medications).transform
     end
 
-    def meds_by_year_and_status_as_string(med_request_data)
-      medication_request_stats_by_year(med_request_data).map do |year, tallies|
-        "- #{year}: #{tallies}"
-      end.join("<br/>\n")
-    rescue
-      'Error formatting medications list!'
+    def med_stats_hash(_form526_submission, assessed_data)
+      { medications_count: assessed_data[:medications]&.size }
```

Comment:
Like number of blood pressure readings, this defines the summary data that we may want to see in the post-EVSS notification. Open to debate on what data we actually want to see.

Reply:
To get a more precise count, we could store medications by year, like https://github.com/department-of-veterans-affairs/vets-api/pull/9580/files#diff-10b3b2429bd9e2e544c255257886ebbea6e0bb9ae0c16f6bf63aaf9499c74bfdL53